File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    answer = 0
    pron = ["aya", "ye", "woo", "ma"]
    for bab in babbling:
        if any(p in bab for p in pron):
            for p in pron:
                bab = bab.replace(p, " ")
        
        if not bab.strip():
            answer += 1
            
    return answer

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        
        String[] babb = {"aya", "ye", "woo", "ma"};
        String tmp = "";
        
        for (String b : babbling) {
            for (String bab : babb) {
                tmp = b.replace(bab, " ");
                b = tmp;
            }
            
            if (b.isBlank()) {
                answer++;
            }   
        }
        return answer;
    }
}